scenarios/crowdsecurity/iptables-scan-multi_ports.yaml

```diff
@@ -1,7 +1,7 @@
 type: leaky
 name: crowdsecurity/iptables-scan-multi_ports
 description: "ban IPs that are scanning us"
-filter: "evt.Meta.log_type == 'iptables_drop' && evt.Meta.service == 'tcp'"
+filter: "evt.Meta.log_type == 'iptables_drop' && evt.Meta.service in ['tcp', 'udp']"
```

Comment:
Since we always hardcoded TCP we did detect UDP drops as well.  Currently I have just replicated the current behaviour but we may want to only look for TCP